src/Wallabag/CoreBundle/Helper/ContentProxy.php

```diff
@@ -146,4 +146,52 @@ private function validateContent(array $content)
     {
         return !empty($content['title']) && !empty($content['html']) && !empty($content['url']);
     }
+
+    /**
+     * Assign some tags to an entry.
+     *
+     * @param Entry        $entry
+     * @param array|string $tags          An array of tag or a string coma separated of tag
+     * @param array        $entitiesReady Entities from the EntityManager which are persisted but not yet flushed
+     *                                    It is mostly to fix duplicate tag on import @see http://stackoverflow.com/a/7879164/569101
+     */
+    public function assignTagsToEntry(Entry $entry, $tags, array $entitiesReady = [])
```

Comment:
Well I blindly rebased my work without seeing that the whole method was missing...